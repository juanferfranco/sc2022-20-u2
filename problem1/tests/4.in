add
ev1
add
ev2
add
ev3
rm
ev1
ask
exit
