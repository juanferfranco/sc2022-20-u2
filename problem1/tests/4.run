./main < tests/4.in
