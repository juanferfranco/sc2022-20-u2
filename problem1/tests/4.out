ev2
ev3
